cloud: optimize 2 docs according to usability tests (#10561)
pingcap/docs: tidb-cloud/manage-user-access.md

> 1. Navigate to the TiDB Cloud login page: <https://tidbcloud.com>.

Reviewer: ```suggestion
1. Go to the TiDB Cloud login page: <https://tidbcloud.com>.
```

2. On the TiDB Cloud login page, sign in with Google, GitHub, or email.

## Sign out

> To sign out from TiDB Cloud, click your account name and select Logout.

Reviewer: ```suggestion
In the upper-right corner of the TiDB Cloud console, click your account name and select **Logout**.
```

There are four roles in an organization. The permissions of each role are as follows:

> 

Author: @langyuezhang PTAL

|                                                                                       | Owner | Member | Billing Admin | Audit Admin |
|---------------------------------------------------------------------------------------|-------|--------|---------------|-------------|
| Invite members to join the organization, and remove members from the organization | ✅     | ❌      | ❌             | ❌           |
| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |

> | Join a project                                                                        | ✅    | ✅      | ✅             | ✅           |

Author: ```suggestion
```

> | Join a project                                                                        | /     | ✅      | ✅             | ✅           |

Author: ```suggestion
| Join a project                                                                        | ✅    | ✅      | ✅             | ✅           |
```

Author: "Join a project" and "Obtain project instance management rights" are not stated for the **owner** role. I'm not sure if they apply to owner.

Reviewer: I think it is both yes. Owners can join a project owned by others and obtain project instance management rights as well. It seems that we also need a technical review of this table.

| View and configure audit logging                                                      | ❌     | ❌      | ❌             | ✅           |

> | Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |

Author: @langyuezhang PTAL

> | Obtain project instance management rights                                             | /     | ✅      | ✅             | ✅           |

Author: ```suggestion
| Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |
```

There are four roles in an organization. The permissions of each role are as follows:

> |                                                                                       | Owner | Member | Billing Admin | Audit Admin |

Author: ```suggestion
|  Permission                                                                        | Owner | Member | Billing Admin | Audit Admin |
```

|---------------------------------------------------------------------------------------|-------|--------|---------------|-------------|
| Invite members to join the organization, and remove members from the organization | ✅     | ❌      | ❌             | ❌           |
| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |
| View and configure audit logging                                                      | ❌     | ❌      | ❌             | ✅           |

> | Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |

Author: ```suggestion
| Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |
| Create API key                                                                                              | ✅     | ❌      | ❌             | ❌            |
```

| Invite members to join the organization, and remove members from the organization | ✅     | ❌      | ❌             | ❌           |
| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |
| View and configure audit logging                                                      | ❌     | ❌      | ❌             | ✅           |
| Obtain project instance management rights                                             | ✅     | ✅      | ✅             | ✅           |

> | Create an API key                                                                                              | ✅     | ❌      | ❌             | ❌            |

Author: ```suggestion
| Manage an API key                                                                                              | ✅     | ❌      | ❌             | ❌            |
```
